system/jlib/jfile.cpp

```diff
@@ -4102,21 +4102,25 @@ IFile * createIFile(const char * filename)
     IFile *ret = createContainedIFileByHook(filename);
     if (ret)
         return ret;
-    bool linremote=(memcmp(filename,"//",2)==0);
-    if (!linremote&&(memcmp(filename,"\\\\",2)!=0)) // see if remote looking
-        return new CFile(filename);
+
     RemoteFilename rfn;
     rfn.setRemotePath(filename);
+
     if (rfn.isNull())
         throw MakeStringException(-1, "CreateIFile cannot resolve %s", filename);
-    if (rfn.isLocal()) { // ignore dafilesrv request if local and standard port
-        StringBuffer tmplocal;
-        return new CFile(rfn.getLocalPath(tmplocal).str());
-    }
+
+    ret = createIFileByHook(rfn);           // use daliservix in preference
+    if (ret)
+        return ret;
+
+    // NB: This is forcing OS path access if not a url beginning '//' or '\\'
+    bool linremote=(memcmp(filename,"//",2)==0);
+    if (!linremote&&(memcmp(filename,"\\\\",2)!=0)) // see if remote looking
+        return new CFile(filename);
+
 #ifdef _WIN32
     StringBuffer tmplocal;
     if (linremote||(rfn.queryEndpoint().port!=0)) {
-        ret = createIFileByHook(rfn);           // use daliservix in preference
         if (ret) 
```

Comment:
What is ret here ?

Reply:
well spotted, harmless, but these 2 lines should be removed.